modules/apps/jax-rs/mdm-admin/src/main/java/org/wso2/carbon/mdm/api/User.java

```diff
@@ -484,19 +485,28 @@ private String getClaimValue(String username, String claimUri) throws MDMAPIExce
      * @param username Username of the user
      * @throws MDMAPIException, UserStoreException, DeviceManagementException
      */
-    private void inviteNewlyAddedUserToEnrollDevice(String username) throws
+    private void inviteNewlyAddedUserToEnrollDevice(String username,String password) throws
                                                                      MDMAPIException, UserStoreException,
                                                                      DeviceManagementException {
         if (log.isDebugEnabled()) {
             log.debug("Sending invitation mail to user by username: " + username);
         }
+        String tennentDomain = CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+        if (tennentDomain.equalsIgnoreCase("carbon.super")) {
```

Comment:
Recommended fixes can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/166